packages/esm-patient-forms-app/src/index.ts

```diff
@@ -38,17 +38,11 @@ function setupOpenMRS() {
   defineConfigSchema(moduleName, {});
 
   return {
-    pages: [
-      {
-        load: getAsyncLifecycle(() => import('./forms/forms-see-all'), options),
-        route: /^chart\/patient\/.+\/forms/,
-      },
-    ],
     extensions: [
       {
         id: 'forms-widget',
         slot: 'patient-chart-summary-dashboard-slot',
-        load: getAsyncLifecycle(() => import('./forms/forms.component'), options),
+        load: getAsyncLifecycle(() => import('./forms/forms-see-all'), options),
```

Comment:
This ticket has an earlier PR ( #314 ) and most of these changes/ new files come from there. Now, the requirement has changed but I have not discarded the changes made in the earlier PR. Some of the files are not required anymore including `forms-see-all`  and the changes can be made to `forms.component` itself.

Reply:
I will make the changes suggested and make a new commit. Thanks.

Reply:
Alright, awesome, thank you! 🥂 
Let me know if you have any questions/run into issues with the suggestions above.

Reply:
`forms.component.tsx` is no longer needed but I haven't yet deleted it just in case we need to refer to it or use it in some way. If necessary I can delete it and make another commit.

Reply:
@NischithJShetty That's fine with me - I must work with this MF next week anyway to adapt it to the recent offline mode changes and can use that chance to make a cleanup if necessary.
Thanks a lot for your work!